src/compiler/types.ts

```diff
@@ -1377,6 +1379,7 @@ module ts {
         /* @internal */ exportSymbol?: Symbol;  // Exported symbol associated with this symbol
         valueDeclaration?: Declaration;         // First value declaration of the symbol
         /* @internal */ constEnumOnlyModule?: boolean; // True if module contains only const enums or other modules with only const enums
+        isAnonymous?: boolean;                  // True if declaration is anonymous

```

Comment:
@mhegazy I was trying a lot of different ways to pinpoint anonymous declarations. I just thought this was easier. But you're right, we shouldn't add something just to serve one very specific use case.

It would be good if you could help me out on the below section. Where I use this property to add a `name` of a declaration/symbol.